network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -374,15 +374,29 @@ class BanneredCampaignPage(PrimaryPage):
     tags = ClusterTaggableManager(through=BanneredCampaignTag, blank=True)
     aside = StreamField(aside_fields, blank=True)
 
+    use_intro_from_parent = models.BooleanField(
+        default=False,
+        blank=False,
+        help_text='This field will overwrite the intro field above with the intro field from this pages parent'
```

Comment:
Since this field does not contain the override but rather a toggle, what do you think about the following change?

```suggestion
        help_text='Overwrite the intro field above with the intro field from this pages parent'
```